_content/essays/world-buddhist-missionary-tour_mahasi.md

```diff
@@ -0,0 +1,19 @@
+---
+title: "World Buddhist Missionary Tour"
+authors:
+  - mahasi
+external_url: "https://static.sirimangalo.org//mahasi/World%20Buddhist%20Missionary%20Tour.htm"
+drive_links:
+  - "https://drive.google.com/file/d/1PvgahHJNE9z7uNqMs30gYQLwf_N9Oiif/view?usp=drive_link"
+formats: [epub]
+course: burmese
+tags:
+  - path
+year: 1995
+month: jun
+pages: 10
+---
+
+> I urge you to meditate beforehand, i.e., before you come across old age, sickness and death, as it is the very time you should make effort to practise Vipassanā insight meditation. I mean you should make full use of time so that you can get one of the said five advantages delivered by Buddha for your own sake.
+
+The brief talk given by Venerable Mahasi Sayadaw discusses the teachings of Vipassana meditation and the importance of Dhamma Thiti Ñāṇa and Nibbāna Ñāna. It emphasizes the precepts and practicing meditation to eliminate mental defilements (kilesās), attain wisdom, and ultimately reach nibbāna, which is liberation from the cycle of suffering.
```

Comment:
Six talks.

Maybe add a link to this history of the tour as well: http://www.myanmarnet.net/nibbana/mahatour.htm

To give people some idea of the background